app/javascript/controllers/map_controller.js

```diff
@@ -9,7 +9,7 @@ export default class extends Controller {
   }
  #addMarkersToMap() {
     this.markersValue.forEach((marker) => {
-      const popup = new mapboxgl.Popup().setHTML(marker.info_window)
+      const popup = new mapboxgl.Popup({focusAfterOpen: false}).setHTML(marker.info_window)
```

Comment:
Removed the focus on open attribute. This makes sure that the first element isn't focussed on when you open the popup (which was causing the blue border surrounding the 'X' close button.